sympy/physics/mechanics/functions.py

```diff
@@ -723,3 +723,118 @@ def flist_iter():
         unzip = lambda l: list(zip(*l)) if l[0] else [(), ()]
         vel_list, f_list = unzip(list(flist_iter()))
     return vel_list, f_list
+
+def Force(force, body1, body2=None, point1=None, point2=None):
```

Comment:
Functions names should follow PEP8 recommendations for capitalization and ideally be a verb not a noun.

Reply:
Corrected!